cmd/ocitools/generate.go

```diff
@@ -58,6 +58,19 @@ var generateFlags = []cli.Flag{
 	cli.StringSliceFlag{Name: "seccomp-errno", Usage: "specifies syscalls to be added to list that returns an error"},
 	cli.StringFlag{Name: "template", Usage: "base template to use for creating the configuration"},
 	cli.StringSliceFlag{Name: "label", Usage: "add annotations to the configuration e.g. key=value"},
+	cli.Uint64Flag{Name: "cpu-shares", Usage: "CPU shares (relative weight (ratio) vs. other cgroups with cpu shares)"},
```

Comment:
Probably update these descriptions to match the man page which matches the runtime-spec Markdown.